src/tab/Tab.tsx

```diff
@@ -0,0 +1,48 @@
+import "./_tab.scss";
+
+import React, {useState} from "react";
+import classNames from "classnames";
+
+import Button from "../button/Button";
+
+export type TabItem = {
+  id: string;
+  children: React.ReactNode;
+  icon?: React.ReactNode;
+};
+
+interface TabProps {
+  items: TabItem[];
+  children: React.ReactNode[];
+  customClassName?: string;
+}
+
+function Tab({items, children, customClassName}: TabProps) {
+  const [activeTabIndex, setActiveTabIndex] = useState(0);
+  const tabClassName = classNames("tab", customClassName);
+
+  return (
+    <div className={tabClassName}>
+      <header className={"tab-header"}>
+        {items.map((item, index) => (
+          <Button
+            key={item.id}
+            customClassName={"tab-header-item"}
+            onClick={handleChangeActiveTab(index)}>
```

Comment:
This looks odd to me, at first glance I thought this was a bug (function called inside `onClick` prop) 😄  I know it is against our `eslint` rules, but I think in this case inline arrow functions can be more readable. And it should be same as current one in performance aspect (which is the main concern of `no-bind` rule)
Or, of course, we can split tab button to its own component and declare event handler in there. What do you think?

Reply:
I think it would be the best to define `TabHeaderItem` component